.github/workflows/ci.yml

```diff
@@ -116,6 +119,29 @@ jobs:
     - name: Run migrations
       run: ./tobira db migrate --config util/dev-config/config.toml
 
+    # UI tests
+    - name: Start docker containers
+      working-directory: util/containers
+      run: |
+        docker-compose -f docker-compose.yml up -d \
+          tobira-auth-proxy \
+          tobira-login-handler \
+          tobira-meilisearch
+
+    - name: Rebuild search index
+      working-directory: backend
+      run: echo yes | cargo run -- search-index update
+
+    - name: Run playwright tests
+      working-directory: frontend
+      run: npx playwright test
+    - name: Upload test results
+      if: always()
+      uses: actions/upload-artifact@v3
+      with:
+        name: playwright-report
+        path: frontend/playwright-report/
+        retention-days: 7
```

Comment:
Oh! I didn't know you can specify retention days here! That could be a solution to our other CI problem. Will think about it again!

Reply:
So yeah I think we should do the following: `ci.yaml` should mark its artifact (not this playwright one, but the main Tobira one) with `retention-days: 1`. Then `deploy.yml` can download it as usually. Well actually, we can simplify that script as we don't need to use custom JS code anymore, but can just use a standard action.
And then, playwright testing should be moved to a new CI job that runs after this `main` job and also downloads the artifact. 
With only 1 day of retention, we shouldn't run into any limit problems.

But I'd suggest doing all of this as a new PR after this one is merged.